lib/beaker/dsl/install_utils/pe_utils.rb

```diff
@@ -483,6 +485,63 @@ def do_install hosts, opts = {}
           end
         end
 
+        # Builds the agent_only and not_agent_only arrays needed for installation
+        # as well as setting the agent_only_check_needed flag.  Note that it
+        # needs to know whether the check is needed to determine which hosts go
+        # in each array.
+        #
+        # @param [Array<Host>]  hosts   hosts to split up into the arrays
+        # @param [String]       pe_ver  overall pe_ver option, will be used if
+        #                               set rather than the hosts pe_ver
+        #
+        # @api private
+        # @return [Boolean, Array<Host>, Array<Host>] whether a version (overall
+        #   overall or on a host) is over 3.99, so agent_only is possible;
+        #   the array of hosts to do an agent_only install on;
+        #   the array of hosts to do our usual install methods on
+        def create_hosts_agent_only_specified_arrays_helper(hosts, pe_ver)
```

Comment:
this is an install utils file, not a helpers file. Between this and the `@api private`, I was trying to denote that this shouldn't be called by a user or from the DSL.

Let me know if you still want me to remove that, and I can do that.